src/decode.ts

```diff
@@ -95,39 +129,31 @@ export function deserialize(qr: string): DataModel {
 			payment.bankAccounts.push(account);
 		}
 
-		data.shift(); // StandingOrderExt
-		data.shift(); // DirectDebitExt
```

Comment:
I changed the logic there. I believe DirectDebitExt should not be shifted unless we know for sure, that data are not a Standing order

Reply:
I revealed it when I was unable to decode an encoded payment. A test helped me to find it out.